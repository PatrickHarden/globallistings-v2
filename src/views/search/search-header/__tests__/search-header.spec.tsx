import React from 'react';
import { shallow } from 'enzyme';

import SearchHeader from '../search-header';

describe('components/SearchComponents', () => {
  describe('SearchHeader', () => {
    it('should render correctly', () => {
      const props = {
        searchHeaderLinkText: 'Go to CBRE',
        searchHeaderLinkUrl: 'https://www.cbre.com',
        searchHeaderTitleText: 'Search Header',
      };
      expect(shallow(<SearchHeader {...props} />)).toMatchSnapshot();
    });

    it('should render correctly with no props', () => {
      expect(shallow(<SearchHeader />)).toMatchSnapshot();
    });

    it('should render correctly with no link', () => {
      expect(shallow(<SearchHeader searchHeaderTitleText="Search Header" />)).toMatchSnapshot();
    });
  });
});
