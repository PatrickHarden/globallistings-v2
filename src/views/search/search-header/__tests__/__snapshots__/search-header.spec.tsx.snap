// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/SearchComponents SearchHeader should render correctly 1`] = `
<styled.div>
  <styled.h4>
    Search Header
  </styled.h4>
  <styled.a
    href="https://www.cbre.com"
  >
    Go to CBRE
  </styled.a>
</styled.div>
`;

exports[`components/SearchComponents SearchHeader should render correctly with no link 1`] = `
<styled.div>
  <styled.h4>
    Search Header
  </styled.h4>
</styled.div>
`;

exports[`components/SearchComponents SearchHeader should render correctly with no props 1`] = `""`;
